foreman/data_refinery_foreman/surveyor/external_source.py

```diff
@@ -31,7 +31,7 @@ class PipelineEnums(Enum):
 
 class ProcessorPipeline(PipelineEnums):
     """Pipelines which perform some kind of processing on the data."""
-    MICRO_ARRAY_TO_PCL = "MICRO_ARRAY_TO_PCL"
+    AFFY_TO_PCL = "AFFY_TO_PCL"
```

Comment:
So this just to narrow it down to just AFFY platforms instead of any type of microarray?